### DISCUSSION TITLE
[Accelerated Tables][Serialisation] datatable_schema  fails to load linked tables*

### DISCUSSION THREAD

**Discussion by lewysigns** [anvilistas/anvil-extras#349] [Q&A] [answered]
### What I am trying to do

I am attempting to use serialization library on the new accelerated tables with a linked column.

### What is happening

This is causing a ```AnvilWrappedError``` in my development-production application that looks like the following:

```
AnvilWrappedError: 'NoneType' object has no attribute 'get_id'
at anvil_extras/serialisation.py, line 110
called from /usr/local/lib/python3.7/site-packages/marshmallow/fields.py, line 1727
called from /usr/local/lib/python3.7/site-packages/marshmallow/fields.py, line 1713
called from /usr/local/lib/python3.7/site-packages/marshmallow/fields.py, line 317
called from /usr/local/lib/python3.7/site-packages/marshmallow/schema.py, line 520
called from /usr/local/lib/python3.7/site-packages/marshmallow/schema.py, line 516
called from /usr/local/lib/python3.7/site-packages/marshmallow/schema.py, line 516
called from /usr/local/lib/python3.7/site-packages/marshmallow/schema.py, line 556
```

The failure occurs when *any* row in a column of a data table is None, IF that column type is a single linked datatable row column.

### Clone 

This started to occurs when the accelerated tables is activated in the data tables setting. I attempted to mimic this in a simple clone below:

https://anvil.works/build#clone:TX5IJADW5BLXJTRB=IGU6N4GJMVJGZXZHMS4NRCBH|TGQCF3WT6FVL2EM2=|C6ZZPAPN4YYF5NVJ=

But it isn't exactly raising the error above. However, if you toggle the accelerated tables setting, you will see that accelerated tables causes the entire linked column NOT to load. This behavior is the root of my issue.

### Question

Has anyone been able to utilize serialisation successfully with the new accelerated tables (with linked rows).

Thanks!

**Comment by s-cork** [comment, marked as answer]
Thanks for reporting - i moved this to #350 and this should get fixed on the development branch later
